src/idl/src/parser.c

```diff
@@ -1,8 +1,8 @@
-/* A Bison parser, made by GNU Bison 3.8.2.  */
+/* A Bison parser, made by GNU Bison 3.5.1.  */
```

Comment:
Using an older bison version probably doesn't cause any issues, but to be sure we should find out what the minimum version is that is required. As this version causes quite some changes in the resulting c code, I think it is safer to use a more recent version.

Reply:
A good point in principle, but we do know 3.5.1 is also fine. Maybe we should agree on a preferred version of bison and try to use that, not because of compatibility concerns, but rather to reduce the amount of noise in the history.

That probably means everyone needs to have a custom version of bison hanging around though, because there are so many Linux distributions (and other platforms) in use, and the system-provided one can be anything.

For now, I wouldn't worry about this.